types/node/index.d.ts

```diff
@@ -756,20 +756,23 @@ declare module "http" {
         write(str: string, encoding?: string, cb?: Function): boolean;
         write(str: string, encoding?: string, fd?: string): boolean;
 
-        writeContinue(): void;
-        writeHead(statusCode: number, reasonPhrase?: string, headers?: any): void;
-        writeHead(statusCode: number, headers?: any): void;
-        statusCode: number;
-        statusMessage: string;
+        addTrailers(headers: any): void;
+        finished: boolean;
+        getHeader(name: string): string;
+        getHeaderNames(): string[]
+        getHeaders(): any
```

Comment:
really? To my understanding ServerResponseHeaders is just an interface. Where is it stated that it inherts from Object.